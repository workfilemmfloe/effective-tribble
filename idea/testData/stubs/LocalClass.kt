class A
trait T
fun foo() {
    class Test : A(), T
}