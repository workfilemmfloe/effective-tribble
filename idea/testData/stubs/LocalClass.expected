PsiJetFileStubImpl[package=]
  CLASS:PsiJetClassStubImpl[name=A fqn=A superNames=[]]
  CLASS:PsiJetClassStubImpl[trait name=T fqn=T superNames=[]]
  FUN:PsiJetFunctionStubImpl[top fqName=foo name=foo]
    VALUE_PARAMETER_LIST:PsiJetParameterListStubImpl
    CLASS:PsiJetClassStubImpl[local name=Test fqn=null superNames=[AT]]
